src/services/completions.ts

```diff
@@ -540,11 +546,15 @@ namespace ts.Completions {
         let isMemberCompletion: boolean;
         let isNewIdentifierLocation: boolean;
         let keywordFilters = KeywordCompletionFilters.None;
+        const useFullSymbolNames = isRightOfCase;
         let symbols: Symbol[] = [];
 
         if (isRightOfDot) {
             getTypeScriptMemberSymbols();
         }
+        else if (isRightOfCase) {

```

Comment:
Wouldn't this restrict completions to *only* enum completions and prevent completions for locals? That is too restrictive.